src/test/java/com/spotify/sparkey/extra/ReloadableSparkeyReaderTest.java

```diff
@@ -54,25 +54,21 @@ private static File createLogFile(String key, String value) throws IOException {
 
   @Test
   public void testFromLogFile() throws ExecutionException, InterruptedException, IOException {
-    final ReloadableSparkeyReader reader = ReloadableSparkeyReader.fromLogFile(logFile1, executorService).get();
-    try {
+    try (ReloadableSparkeyReader reader = ReloadableSparkeyReader.fromLogFile(logFile1, executorService)
+            .toCompletableFuture().get()) {
```

Comment:
Use deterministic or single threaded executor in tests so that you can use `.getNow()` instead of the blocking `.get()`

Reply:
Good idea, but will leave it for now.